packages/isomorphic/src/utils/shared.ts

```diff
@@ -0,0 +1,41 @@
+import { concatBytes } from '@noble/hashes/utils'
+
+export function concat(views: Uint8Array[]): Uint8Array {
+  return concatBytes(...views)
+}
+
+//
+// export function concat(views: ArrayBufferView[]) {
```

Comment:
Haven't decided if this util should be used always or if i needs to have separate node and browser implementations.